ffmpeg/README.md

```diff
@@ -0,0 +1,119 @@
+    zlib1g-dev
+    ```
+1. On the virtual machine, clone and cd into the [ffmpeg GitHub repository](https://github.com/ffmpeg/ffmpeg) by running
+
+    ```
+    git clone https://github.com/ffmpeg/ffmpeg && \
+    cd ffmpeg
+    ```
+1. Checkout the release you want to compile by running
+
+    ```
+    git fetch --all --tags --prune && \
+    git checkout tags/<tag_name> -b <branch_name>
+    ```
+    where `<tag_name>` is the name of the tag you want to checkout, and `<branch_name>` is the name of a new local branch to checkout that commit on.
+1. On the virtual machine, configure the source for compilation & make by running
+
+    ```
+    ./configure \
+    --disable-shared \
+    --enable-static \
+    --pkg-config-flags="--static" \
+    --extra-cflags="-static" \
+    --extra-ldflags="-static" \
+    --enable-version3 && \
+    make
+    ```
+1. The root directory of the ffmpeg repo will now have the desired binary `ffmpeg`.
+
+### Windows:
+
+1. [Install Vagrant](https://www.vagrantup.com/downloads.html).
+1. Clone and cd into this [Windows Vagrant GitHub repository](https://github.com/oconnormi/vagrant-windows) made by Michael O'Connor by running
+
+    ```
+    git clone https://github.com/oconnormi/vagrant-windows
+    ```
+1. Run `vagrant up`.
+1. Run `vagrant rdp`.
+1. On the virtual machine, [download MSYS2 32 bit](http://repo.msys2.org/distrib/i686/msys2-i686-20161025.exe).
+1. On the virtual machine, start `mingw32.exe` found under `C:\msys32`. This will be our terminal for the remainder of this guide.
+1. On the virtual machine, install compilation dependencies by running
+
+    ```
+    pacman -S make \
+    diffutils \
+    yasm \
+    mingw-w64-i686-gcc
+    ```
+1. On the virtual machine, clone and cd into the [ffmpeg GitHub repository](https://github.com/ffmpeg/ffmpeg) by running
+
+    ```
+    git clone https://github.com/ffmpeg/ffmpeg && \
+    cd ffmpeg
+    ```
+Note that mingw32 doesn't have git installed. You can open `Git Bash` to use git, but make sure to switch back to mingw32 for the remainder of this guide.
+1. Checkout the release you want to compile by running
+
+    ```
+    git fetch --all --tags --prune && \
+    git checkout tags/<tag_name> -b <branch_name>
+    ```
+    where `<tag_name>` is the name of the tag you want to checkout, and `<branch_name>` is the name of a new local branch to checkout that commit on.
+1. On the virtual machine, configure the source & make by running
+
+    ```
+    ./configure \
+    --disable-shared \
+    --enable-static \
+    --pkg-config-flags="--static" \
+    --extra-cflags="-static" \
+    --extra-ldflags="-static" \
+    --enable-version3 && \
+    make
+    ```
+1. The root directory of the cloned ffmpeg repo will now have the desired binary `ffmpeg.exe`.
+
+### Testing
+To test the binary run the following command
+```
+./ffmpeg -i <MPG_FILE> -vf thumbnail,scale=200:-1 -frames:v 1 -vsync vfr thumbnail_%d.png
+```
+Console output and a thumbnail means it worked.
```

Comment:
✏️ `If successful, there should be console output and a thumbnail named "thumbnail_#.png" in the current directory.